package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "wp-job-manager",
-  "version": "1.39.0",
+  "version": "1.40.0",
```

Comment:
I think we need to run an `npm install` to also update the `package-lock`.

Reply:
Done! [a28122f](https://github.com/Automattic/WP-Job-Manager/pull/2413/commits/a28122f7bc329835219f15d905c43f8d484730b4)